CMakeLists.txt

```diff
@@ -392,6 +416,8 @@ find_package(Boost 1.49
       program_options
       unit_test_framework
       serialization
+      system
+      filesystem
```

Comment:
I'm wondering if we could use C++11 filesystem support to do the same, boost dependencies was always a critic point.

Reply:
Yes, I am finding difficulties regarding the same.

Reply:
Also, the Jenkins build would fail.

Reply:
I think we installed every boost lib, so the Jenkins build is probably fine. But ideally, we can remove the boost filesystem dependency with a C++11 standard function.

Reply:
It looks like we need boost filesystem for loading images from a folder, what about we could check for ``std::experimental::filesystem`` or ``std::filesystem`` and provide the functionality only if the feature is supported. Let me know what you think.

Reply:
Okay sure